projects/js-packages/shared-extension-utils/package.json

```diff
@@ -17,6 +17,7 @@
 		"test": "jest"
 	},
 	"dependencies": {
+		"@automattic/color-studio": "2.5.0",
```

Comment:
It's a bit annoying that we have to bring that whole dependency just for a single color. Part of me would want to just hardcode the color like we do in other places like here:
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/js-packages/components/components/jetpack-logo/index.tsx#L7

What do you think?

Reply:
I'm okay with that.